tests/cnf/core/network/sriov/tests/qinq.go

```diff
@@ -663,3 +814,130 @@ func disableQinQOnSwitch(switchCredentials *sriovenv.SwitchCredentials, switchIn
 
 	return nil
 }
+
+func defineTestServerPmdCmd(ethPeer, pciAddress string) []string {
+	baseCmd := fmt.Sprintf("dpdk-testpmd -a %s -- --forward-mode txonly --eth-peer=0,%s "+
+		"--cmdline-file=/etc/cmd/cmd_file", pciAddress, ethPeer)
+
+	baseCmd += " --stats-period 5"
+
+	return []string{"/bin/bash", "-c", baseCmd}
+}
+
+func defineTestClientPmdCmd(ethPeer, pciAddress string) []string {
+	baseCmd := fmt.Sprintf(
+		"timeout -s SIGKILL 20 dpdk-testpmd --vdev=virtio_user0,path=/dev/vhost-net,iface=net2 -a %s -- "+
+			"--forward-mode txonly --eth-peer=0,%s", pciAddress, ethPeer)
+
+	baseCmd += " --stats-period 5"
+
+	return []string{baseCmd}
+}
+
+func defineAndCreateServerDPDKPod(
+	podName,
+	nodeName string,
+	serverPodNetConfig []*multus.NetworkSelectionElement,
+	podCmd []string) *pod.Builder {
+	var rootUser int64 = 0
+	securityContext := corev1.SecurityContext{
+		RunAsUser: &rootUser,
+		Capabilities: &corev1.Capabilities{
+			Add: []corev1.Capability{"IPC_LOCK", "SYS_RESOURCE", "NET_RAW"},
+		},
+	}
+
+	dpdkContainer := pod.NewContainerBuilder(podName, NetConfig.DpdkTestContainer, podCmd)
+	dpdkContainerCfg, err := dpdkContainer.WithSecurityContext(&securityContext).
```

Comment:
Can we?
```
dpdkContainerCfg, err :=pod.NewContainerBuilder(podName, NetConfig.DpdkTestContainer, podCmd).WithSecurityContext(&securityContext).WithResourceLimit("2Gi", "1Gi", 4).
		WithResourceRequest("2Gi", "1Gi", 4).WithEnvVar("RUN_TYPE", "testcmd").
		GetContainerCfg()
```

Reply:
Please remove `pod.NewContainerBuilder(podName, NetConfig.DpdkTestContainer, podCmd)` and implement it as shown above.

Reply:
I dont understand the difference. Looks the same. I will ping you.